### PR TITLE
ref(ui): Convert robot to emotion

### PR DESCRIPTION
**Pull request by scttcper**
part of [WOR-307](https://getsentry.atlassian.net/browse/WOR-307)

### CODE REVIEW

File: src/sentry/static/sentry/app/components/errorRobot.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import withApi from 'app/utils/withApi';
 import {Client} from 'app/api';
 import {LightWeightOrganization, Project} from 'app/types';
 import {defined} from 'app/utils';
+import space from 'app/styles/space';
+import robotBackground from 'app/../images/spot/sentry-robot.png';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/errorRobot.tsx:13]: Is `errorRobot` lazy loaded?

File: src/sentry/static/sentry/app/components/errorRobot.tsx [hunk 2]
```diff
@@ -103,19 +105,14 @@ class ErrorRobot extends React.Component<Props, State> {
...
           <h3>Waiting for events…</h3>
           <p>
-            <span>
-              <span>Our error robot is waiting to </span>
-              <span className="strikethrough">
-                <span>devour</span>
-              </span>
-              <span> receive your first event.</span>
-            </span>
+            Our error robot is waiting to <Strikethrough>devour</Strikethrough> receive
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/errorRobot.tsx:114]: Can we use `t()` here and in the `<h3>`

File: src/sentry/static/sentry/app/components/errorRobot.tsx (hunk 1 above)
Comment (scttcper) [reply src/sentry/static/sentry/app/components/errorRobot.tsx:13]: I've added lazy loading to the robot, please double check

File: src/sentry/static/sentry/app/views/issueList/noGroupsHandler/index.tsx [hunk 3]
```diff
@@ -115,13 +114,19 @@ class NoGroupsHandler extends React.Component<Props, State> {
...
+    );
+
     return (
-      <ErrorRobot
-        org={organization}
-        project={project}
-        sampleIssueId={sampleIssueId?.toString() ?? ''}
-        gradient
-      />
+      <React.Suspense fallback={null}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/issueList/noGroupsHandler/index.tsx:122]: Do we need a `<Placeholder/>` here?

**Review by billyvg — APPROVED** [review]

**Pull request merged**